### ISSUE TITLE
finish task loss cannot converge 

### ISSUE THREAD

**Issue by ponpontio** [shenweichen/Bytedance_ICME2019_challenge_baseline#1]
i trained the model with two epochs.
the loss is like:
> 19622340/19622340 [==============================] - 417s 21us/step - loss: 0.5772 - finish_loss: 0.5248 - like_loss: 0.0292

the finish task seems not converging.

**Comment by shenweichen** [comment]
@newtiopon Hi,I think you can check if it is overfitted when training 2 or more epochs
